packages/bimap/bimap.2020Dec27/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "An OCaml library implementing bi-directional maps and multi-maps"
+description:
+  "Bi-directional maps permit clients to not only obtain a value provided a key, but to obtain a key provided a value"
+maintainer: ["papatangonyc@gmail.com"]
+authors: ["papatangonyc@gmail.com"]
+license: "GPLv3"
+homepage: "https://github.com/pat227/bimap.git"
+bug-reports: "https://github.com/pat227/bimap.git/issues"
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "core" {>= "v0.11.3"}
+  "dune" {>= "2.0"}
+  "ounit" {>= "2.2.2"}
```

Comment:
Shouldn't this be a test dependency? (should look something like `with-test & >= "2.2.2"`)

Reply:
Will do and will resubmit.